MANIFEST.in

```diff
@@ -0,0 +1,37 @@
+recursive-include unionml/templates *
+
+include README.md
+include src/pythae/py.typed  # marker file for PEP 561
+
+include CHANGELOG.md
+include changelog.md
+include LICENSE
+include LICENSE.txt
+include CITATION.cff
+include *.cff  # citation info
+
+include MANIFEST.in
+include src/pythae/pyproject.toml
```

Comment:
Not needed if we remove [L6-10](https://github.com/clementchadebec/benchmark_VAE/blob/main/setup.py?plain=1#L6) and corresponding  imports in `setup.py` ([line 1](https://github.com/clementchadebec/benchmark_VAE/blob/main/setup.py?plain=1#L1) and [line 3](https://github.com/clementchadebec/benchmark_VAE/blob/main/setup.py?plain=1#L3))

Reply:
I also thought of this. It's good that we are on the same page here. 

Also, I would suggest you to change the directory structure from `src/pythae` to just `pythae`. This will require a minor modification in setup.py. 

But if you are okay with this change, I am happy to send that in this PR as well.

Reply:
That's cool if you can move the `pyproject.toml` file! As discussed before, we should also keep the `include pyproject.toml` in the MANIFEST.ini. 
Thank you for suggesting changing the structure but I would like to keep it like this at least for now :).

Reply:
Done